### PR TITLE
Copy over Mike's work and add minor installation script updates

### PR DESCRIPTION
**Pull request by Shabirmean**
## What's this?
- This PR does the following:
   - Add images to support the README
   - Fix some of the hardcoded values to use env vars
   -  **Copies over the lates changes from @mike-ensor's Gitlab project into this repository**

### CODE REVIEW

File: anthos-baremetal-edge-deployment/roles/abm-post-install/templates/collectd.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+LoadPlugin processes
+LoadPlugin tcpconns
+LoadPlugin write_gcm
+LoadPlugin match_regex
+LoadPlugin match_throttle_metadata_keys
+LoadPlugin stackdriver_agent
+
+<Plugin "interface">
+  Interface "eno1"
+  Interface "eno1.410"
```
Comment (mike-ensor) [inline anthos-baremetal-edge-deployment/roles/abm-post-install/templates/collectd.conf.j2:54]: You can remove this line and the following (460)...these are specific to VLANs that won't be used

File: anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/setup-vlan-interfaces.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+- name: Print network interfaces
```
Comment (mike-ensor) [inline anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/setup-vlan-interfaces.yml:1]: You can take this file out, and the reference to it in the main.yaml of the same level...this just sets up VLANs which should be a phase 2 or 3

File: anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/main.yml [hunk 3]
```diff
@@ -174,8 +121,34 @@
...
+  - docker
+
 - name: "Setup docker on host machine"
   include: setup-docker-apt.yml
+  when: ( docker_version.stdout == "" or docker_version.stdout is version('20.9.0', '<') )
   tags:
   - docker
-  # - never
+
+- name: "Setup VLAN Interfaces on Physical Hosts"
```
Comment (mike-ensor) [inline anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/main.yml:149]: See comment below, but this section can be taken out

File: anthos-baremetal-edge-deployment/roles/abm-post-install/templates/collectd.conf.j2 (hunk 1 above)
Comment (Shabirmean) [reply anthos-baremetal-edge-deployment/roles/abm-post-install/templates/collectd.conf.j2]: Removed in f7a7516. Confirm it is correct please!

File: anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/setup-vlan-interfaces.yml (hunk 2 above)
Comment (Shabirmean) [reply anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/setup-vlan-interfaces.yml]: Removed in 40d2707. Please confirm it is correct!

File: anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/main.yml (hunk 3 above)
Comment (Shabirmean) [reply anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/main.yml]: Removed in 40d2707

**Comment by Shabirmean** [comment]
@mike-ensor - please TAL at the commits with the changes and ACK if they good please :)

**Comment by mike-ensor** [comment]
LGTM

**Pull request merged**